scripts/apps/vocabularies/controllers/VocabularyEditController.tsx

```diff
@@ -111,6 +126,60 @@ export function VocabularyEditController(
         return true;
     }
 
+    if ($scope.vocabulary.field_type === EDITOR_BLOCK_FIELD_TYPE) {
+        const vocabulary = $scope.vocabulary;
+
+        $scope.item = {
+            fields_meta: {
+
+                /**
+                 * Fake field, needed for compatibility with sdEditor3 directive
+                 */
+                [editorBlockFieldId]: {draftjsState: vocabulary.field_options?.template},
```

Comment:
either set fields meta conditionally(when `vocabulary.field_options.template` is present) or do a fallback - empty content state converted to raw. It should never be
`fields_meta: {"editor_block_field": undefined}`

Reply:
It can possibly only be `{fields_meta: {"editor_block_field": {draftjsState: undefined}}`, that's supported by types. Everything is properly typed here and linter doesnt fail

Reply:
I'm not suggesting the linter fails. I'm saying it's a bad idea to have such object possible.